main.js

```diff
@@ -703,6 +703,15 @@ function main() {
         jsDeclarationServer.provideAmbientDeclarations(tsAmbient);
     } catch (e) {
         adapter.log.warn('Could not read TypeScript ambient declarations: ' + e);
+        // This should not happen, so send a error report to Sentry
+        if (adapter.supportsFeature && adapter.supportsFeature('PLUGINS')) {
+            const sentryInstance = adapter.getPluginInstance('sentry');
+            if (sentryInstance) {
+                sentryInstance.getSentryObject().captureException(e);
```

Comment:
Better verify that getSentryObject is really returning an bject and not null please

Reply:
Can that happen? Better update your documentation then :)